src/core/fetchers/segment/__tests__/prioritizer.test.ts

```diff
@@ -651,14 +651,14 @@ describe("SegmentFetchers Prioritizer", () => {
 
               prioritizer.updatePriority(pObs2, 20 + 12);
               break;
-            case 1:
+            case 1: // interrupting pObs2 now that it has been updated
               expect(evt.type).toEqual("interrupted");
               expect(obs1Started).toEqual(1);
               expect(obs2Started).toEqual(1);
-              expect(obs3Started).toEqual(0);
+              expect(obs3Started).toEqual(1);
```

Comment:
Considering that we already checked whether it was set to `1` in the previous iteration, I don't know how it passed here